src/Mail/Service/MailService.php

```diff
@@ -202,26 +204,24 @@ protected function renderBody(Message $message, array $options, array $variables
             throw new InvalidArgumentException('"template" parameter is missing from options');
         }
 
-        $html = $this->getRenderer()->render($options['template'], $variables);
+        $body = new MimeMessage;
 
-        // We only need to set the HTML view
-        if (!array_key_exists('template_text', $options)) {
-            $message->getHeaders()->addHeaderLine('Content-Type', 'text/html');
-            $message->setBody($html);
-            return;
+        if (array_key_exists('template_text', $options)) {
```

Comment:
I had to switch these around, as we always need a body object(and not just a string)